### PR TITLE
WEB-796 fix safari and media card border radius

### PR DESCRIPTION
**Pull request by sergivillar**

### CODE REVIEW

File: src/card.css.ts [hunk 1]
```diff
@@ -9,7 +9,12 @@ export const actions = style([
     {marginTop: 16},
 ]);
 
-export const boxed = sprinkles({display: 'flex'});
+export const boxed = style([
+    sprinkles({display: 'flex'}),
+    {
+        isolation: 'isolate', // Needed to preserve border-radius with Video component and Safari
```
Comment (sergivillar) [inline src/card.css.ts:15]: this create a new stacking context, similar to the old hack of `translateZ(0)`

**Comment by sergivillar** [comment]
> **Screenshot tests report**
> 
> ❌ carousel / carousel-mobile

I think this test could be updated, WDYT @atabel ?

File: src/card.css.ts (hunk 1 above)
Comment (atabel) [reply src/card.css.ts:15]: oh, and Safari support this? from which version?

**Review by atabel — APPROVED** [review]

**Comment by atabel** [comment]
> > **Screenshot tests report**
> > x carousel / carousel-mobile
> 
> I think this test could be updated, WDYT @atabel ?

to be fair, I don't see the difference, so yes

File: src/card.css.ts (hunk 1 above)
Comment (sergivillar) [reply src/card.css.ts:15]: since safari 8 :O

**Pull request merged**